crates/nu-parser/src/lite_parser.rs

```diff
@@ -377,14 +388,28 @@ fn push_command_to(
     if !command.is_empty() {
         match last_connector {
             TokenContents::OutGreaterThan => {
-                pipeline.push(LiteElement::Redirection(
-                    last_connector_span
-                        .expect("internal error: redirection missing span information"),
-                    Redirection::Stdout,
-                    command,
-                ));
+                let span = last_connector_span
+                    .expect("internal error: redirection missing span information");
+                if pipeline.exists(Redirection::Stdout) {
+                    return Some(ParseError::LabeledError(
+                        "Can't make stdout redirection twice".into(),
```

Comment:
You can use positive text to make it more readable, like:

"Redirection can be set only once"

Reply:
Thanks, it's a good improvement